packages/core/src/spec/lib/features/FeatureManagerSpec.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {FeatureManager} from '../../../lib/features/FeatureManager';
+import {AppsFlyerConfig, AppsFlyerFeature} from '../../../lib/features/AppsFlyerFeature';
+import {TwaManifest} from '../../../lib/TwaManifest';
+
+describe('FeatureManager', () => {
+  describe('#constructor', () => {
+    it('Creates from empty features', () => {
+      const manifest = {
+        features: {},
+        fallbackType: 'customtabs',
+      } as TwaManifest;
+      const features = new FeatureManager(manifest);
+      expect(features.androidManifest.components.length).toBe(0);
```

Comment:
Do you think these would read better as:

```
expect(features.androidManifest.components).toBe([]);
```

Reply:
Done. I think this really shines in this example:

```typescript
      expect(appsFlyerFeature.applicationClass.variables)
          .toEqual(['private static final String AF_DEV_KEY = "12345";']);
```